nasa-gibs/worldview
# WV-2970 fix

## Description
Recently added e2e test is failing in github. 

Fixes #WV-2970

## How To Test
git checkout wv-2970-fix
run tests locally with `npx playwright test classification-test.spec.js`
Confirm GH tests pass 

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
